app/controllers/books_controller.rb

```diff
@@ -3,10 +3,12 @@ class BooksController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @books = Book.all
+    @books = Book.page(params[:page]).per(5)
```

Comment:
if we actually want to hard-coded a `per` params (which normally we don't). we can specify `paginates_per ` on each model.

ref: https://github.com/kaminari/kaminari#configuring-default-per_page-value-for-each-model-by-paginates_per